core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -243,7 +243,7 @@ public boolean equalsAliases(MetaData other) {
      *
      * @param aliases         The names of the index aliases to find
      * @param concreteIndices The concrete indexes the index aliases must point to order to be returned.
-     * @return the found index aliases grouped by index
+     * @return a map of index to a list of alias metadata, the list will be empty if no aliases are present
```

Comment:
Can you clarify this comment? Something like `the list corresponding to a concrete index will be empty if no aliases are present for that index`?